package.json

```diff
@@ -158,11 +158,10 @@
     "postcss-modules-parser": "^1.1.1",
     "postcss-modules-scope": "^2.1.0",
     "postcss-modules-values": "^2.0.0",
-    "postcss-shopify": "^2.2.1",
     "puppeteer": "^1.20.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-test-renderer": "^16.3.1",
+    "react-test-renderer": "^16.8.6",
```

Comment:
Not strictly needed but I though this would help with some test stuff but it didn't. This should be kept in sync with the version of react/react-dom anyway.